lib/cuke_linter/linters/unique_scenario_names_linter.rb

```diff
@@ -0,0 +1,108 @@
+module CukeLinter
```

Comment:
In every linter in `lib/cuke_linter/linters` there is a `message` method. But you are not defining one. Why?

Reply:
linter.rb:

```
    def initialize(name: nil, message: nil, rule: nil)
      @name    = name || self.class.name.split('::').last
      @message = message || "#{self.name} problem detected"
      @rule    = rule
    end
```

Reply:
I generate the message dynamically and reassign the variable for checking scenarios and outline scenarios methods to show specified messages. But also an option make common message

```
def message
  @message = "Scenario name created by Scenario Outline '#{duplicates.first}' is not unique. \n"\
              "    Original name is on line: #{original_line} \n"\
              "    Duplicate is on: #{duplicate_lines}"
end
```